[DOCS][OV JS] Update JS API readme (#23138)
openvinotoolkit/openvino: src/bindings/js/docs/README.md

# OpenVINO™ JavaScript Bindings

## Folders

- `./docs` - documentation
- `./node` - openvino-node npm package

## openvino-node Package Developer Documentation

### Components

- [include](../node/include/) - header files for current API.
- [lib](../node/lib/) - TypeScript sources for current API.
- [src](../node/src/) - C++ sources for current API.
- [tests](../node/tests/) - tests directory for current API.

### Build

> - Make sure that all submodules are updated `git submodule update --init --recursive`

Reviewer: ````suggestion
- Make sure that all submodules are updated:
  ```
  git submodule update --init --recursive
  ```
````

> - Create build dir `mkdir build && cd build`

Reviewer: ````suggestion
- Create the *build* directory:
   ```
   mkdir build && cd build
   ```
````

> - Configure binaries building:

Reviewer: ```suggestion
- Configure building of the binaries:
```

  ```bash
  cmake \
    -DCMAKE_BUILD_TYPE=Release \
    -DENABLE_FASTER_BUILD=ON \
    -DCPACK_GENERATOR=NPM \
    -DENABLE_SYSTEM_TBB=OFF -UTBB* \
    -DENABLE_TESTS=OFF \
    -DENABLE_SAMPLES=OFF \
    -DENABLE_WHEEL=OFF \
    -DENABLE_PYTHON=OFF \
    -DENABLE_INTEL_GPU=OFF \
    -DCMAKE_INSTALL_PREFIX=../src/bindings/js/node/bin \
    ..
  ```
- Build bindings:

>   `cmake --build . --config Release --verbose -j4`

Author: Fix build instruction

> - Install binaries for openvino-node package:
>   `cmake --install .`

Reviewer: ````suggestion
- Install binaries for the *openvino-node* package:
  ```bash
  cmake --install .
  ```
````

Author: Fix install instruction

- Go to npm package folder `cd ../src/bindings/js/node`
- Now you can install dependencies packages and transpile ts to js code. Run `npm install`
- Run tests `npm run test` to make sure that **openvino-node** built successfully

## Usage

> - Add `openvino-node` package in your project, specify in **package.json**: `"openvino-node": "file:*path-to-current-directory*"`
> - Require by: `const ov = require('openvino-node');`

Reviewer: ````suggestion
- Add the **openvino-node** package to your project by specifying it in **package.json**: 
  ```json
  "openvino-node": "file:*path-to-current-directory*"
  ```
- Make sure to require it:
  ```
  const ov = require('openvino-node');
  ```
````
